scripts/itkdev-docker-compose

```diff
@@ -114,6 +122,50 @@ case "$cmd" in
     open $($script_path url)
     ;;
 
+  sync)
+    itkdev-docker-compose sync:db
+    itkdev-docker-compose sync:files
+    ;;
+
+  sync:db)
+    if [ -z "${REMOTE_HOST:-}" ]; then
+      (>&2 echo "${bold}Environment variable REMOTE_HOST not defined in $dot_env_file${normal}")
+      exit 1
+    fi
+    if [ -z "${REMOTE_DB_DUMP_CMD:-}" ]; then
+      (>&2 echo "${bold}Environment variable REMOTE_DB_DUMP_CMD not defined in $dot_env_file${normal} see readme for more information about this setting")
+      exit 1
+    fi
+
+    ssh deploy@${REMOTE_HOST} ${REMOTE_DB_DUMP_CMD} | eval $(itkdev-docker-compose sql:connect)
+
```

Comment:
Extra blank line.

Reply:
I ran into issues with the eval and an error but it seams to work now. So removed the newline.